lib/resources/oracledb_session.rb

```diff
@@ -41,7 +41,7 @@ def initialize(opts = {})
       @db_role = opts[:as_db_role]
 
       # we prefer sqlci although it is way slower than sqlplus, but it understands csv properly
-      @sqlcl_bin = 'sql' unless opts.key?(:sqlplus_bin) # don't use it if user specified sqlplus_bin option
+      @sqlcl_bin = opts[:sqlcl_bin] || 'sql' unless opts.key?(:sqlplus_bin) # don't use it if user specified sqlplus_bin option
```

Comment:
The comment mentions something called sqlci, the option is named sqlcl, and the actual value is either 'sql' or 'sqlplus'.  Could you clarify?  Assume the maintainer of this code is not familiar with Oracle.

Reply:
The comment is incorrect, i've corrected it to `sqlcl`